code/modules/mob/living/carbon/human/human_organs.dm

```diff
@@ -12,15 +12,25 @@
 
 	number_wounds = 0
 	var/force_process = 0
+	var/splint_process = 0
 	var/damage_this_tick = getBruteLoss() + getFireLoss() + getToxLoss()
 	if(damage_this_tick > last_dam)
 		force_process = 1
+	if(splinted_limbs.len)
+		splint_process = 1
 	last_dam = damage_this_tick
 	if(force_process)
 		bad_external_organs.Cut()
 		for(var/obj/item/organ/external/Ex in bodyparts)
 			bad_external_organs |= Ex
 
+	else if(splint_process) //we don't need to process every organ if it's just one limb splinted
```

Comment:
Okay, well, originally I was going to do it that way, but the thought of processing every mob's organs on each step seemed laggy, and then it turned out I  needed a special splint list anyways for the life() method to work properly, so yeah, probably, like Remie was saying.